ci/dhis2-verify-app.yml

```diff
@@ -0,0 +1,164 @@
+              id: yarn-cache
+              with:
+                  path: '**/node_modules'
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install
+              if: steps.yarn-cache.outputs.cache-hit != 'true'
+              run: yarn install --frozen-lockfile
+
+            - name: Build
+              run: yarn build
+
+            - uses: actions/upload-artifact@v2
+              with:
+                  name: app-build
+                  path: '**/build'
+                  retention-days: 1
+
+    lint:
+        runs-on: ubuntu-latest
+        needs: install
+        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: actions/cache@v2
+              id: yarn-cache
+              with:
+                  path: '**/node_modules'
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install
+              if: steps.yarn-cache.outputs.cache-hit != 'true'
+              run: yarn install --frozen-lockfile
+
+            - name: Lint
+              run: yarn lint
+
+    test:
+        runs-on: ubuntu-latest
+        needs: install
+        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: actions/cache@v2
+              id: yarn-cache
+              with:
+                  path: '**/node_modules'
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install
+              if: steps.yarn-cache.outputs.cache-hit != 'true'
+              run: yarn install --frozen-lockfile
+
+            - name: Test
+              run: yarn test
+
+    publish:
+        runs-on: ubuntu-latest
+        needs: [build, lint, test]
+        if: "!contains(github.event.head_commit.message, '[skip ci]')"
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  token: ${{env.GH_TOKEN}}
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: actions/download-artifact@v2
+              with:
+                  name: app-build
+
+            - uses: dhis2/deploy-build@master
+              env:
+                  INPUT_BUILD_DIR: build/app
+              with:
+                  # uncomment following line for monorepo apps:
+                  #cwd: ./packages/app
+                  build-dir: build/app
+                  github-token: ${{ env.GH_TOKEN }}
+
+    release:
+        runs-on: ubuntu-latest
+        needs: [publish]
+        if: "github.event_name == 'push' && !contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
We do not want to create a release in GitHub for feature branches, only on the specified branches.